### ISSUE TITLE
[v6.0.0-beta.30] blank canvas glitch

### ISSUE THREAD

**Issue by Kosai106** [storybookjs/storybook#11206]
**Describe the bug**
I've found a way to trigger a visual glitch which stops the canvas and docs from rendering.

**To Reproduce**
1. Visit a Storybook on v6.0.0-beta.30 (I haven't tested others)
2. Click on `...` and then `About your Storybook`
3. Attempt to visit a story again

**Expected behavior**
The same behavior as `5.3` is expected - To see my stories after clicking this button.

**Screenshots**
<img width="1920" alt="Screenshot 2020-06-16 at 20 07 44" src="https://user-images.githubusercontent.com/6379824/84811382-1153f600-b00d-11ea-8459-faac3bbd34f6.png">

**System:**
```
System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  Binaries:
    Node: 12.13.1 - ~/.nvm/versions/node/v12.13.1/bin/node
    Yarn: 1.13.0 - /usr/local/bin/yarn
    npm: 6.12.1 - ~/.nvm/versions/node/v12.13.1/bin/npm
  Browsers:
    Chrome: 83.0.4103.97
    Firefox: 76.0.1
    Safari: 13.1.1
```

**Comment by shilman** [comment]
cc @ndelangen @kylesuss

**Comment by ndelangen** [comment]
Yeah I can confirm, this is a bug.

@Kosai106 would you be able to help fix this with a PR?

**Comment by Kosai106** [comment]
For sure @ndelangen, I'll try to dig in tonight.

**Comment by ndelangen** [comment]
Awesome, if you need help getting set up, LMK!

**Comment by Tomastomaslol** [comment]
I took a quick look at this and I was able to recreate the behaviour in `v6.0.0-beta.30`, but was unable to recreate it in `6.0.0-rc.28`.

### Behavouir in `v6.0.0-beta.30`
![beta30](https://user-images.githubusercontent.com/1872246/89601091-026c2000-d8a7-11ea-98d0-29843588fa3d.gif)

### Behavouir in `v6.0.0-rc.28`
![rc28](https://user-images.githubusercontent.com/1872246/89601148-2def0a80-d8a7-11ea-87b7-bc4a2664d7e5.gif)


If my understanding of the reported issue is correct I believe this issue can be marked as resolved.

**Comment by ndelangen** [comment]
I'm rather confident this is fixed in the 6.0.0 release, please re-open if you see this still happens.
